packs/journals/archetypes.json

```diff
@@ -2594,7 +2594,7 @@
+                "content": "<ul class=\"tags paizo-style\"><li class=\"tag rarity rare\">Rare</li></ul><p>The source of your magic buckles against your control, always pressing to be released.</p><p><strong>Additional Feats:</strong> <strong>8th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Energetic Resonance], @UUID[Compendium.pf2e.feats-srd.Item.Spell Relay]; <strong>10th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Surging Might]; <strong>16th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Scintillating Spell]; <strong>18th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Echoing Spell]</p><h2>@UUID[Compendium.pf2e.classfeatures.Item.Wellspring Magic] <span style=\"float:right\">1st</span></h2><p>@Embed[Compendium.pf2e.classfeatures.Item.mRvyq7G0rqRP1EAr inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Wellspring Mage Dedication] <span style=\"float:right\">Level 2</span></h2><ul class=\"tags paizo-style\"><li class=\"tag rarity rare\">Rare</li><li class=\"tag traits\">archetype</li><li class=\"tag traits\">class</li><li class=\"tag traits\">dedication</li></ul><p><strong>Prerequisites</strong> wellspring magic</p><hr /><p>There's no longer a limit on how many temporary spell slots you can gain per day from wellspring magic.</p><p><strong>Special</strong> You can't select another dedication feat until you gain two other feats from the @UUID[Compendium.pf2e.journals.JournalEntry.vx5FGEG34AxI2dow.JournalEntryPage.zj9z2BwVX6TLHdx3]{Wellspring Mage} archetype.</p><p><span style=\"float:right\"><em>Source: Pathfinder Secrets of Magic</em></span></p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Wellspring Control] <span style=\"float:right\">Level 4</span></h2><ul class=\"tags paizo-style\"><li class=\"tag traits\">archetype</li><li class=\"tag traits\">fortune</li></ul><p>@Embed[Compendium.pf2e.feats-srd.Item.7nnEtLdUKmDljdhm inline]</p><p><span style=\"float:right\"><em>Source: Pathfinder Secrets of Magic</em></span></p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Urgent Upwelling] <span style=\"float:right\">Level 6</span></h2><ul class=\"tags paizo-style\"><li class=\"tag traits\">archetype</li></ul><p>@Embed[Compendium.pf2e.feats-srd.Item.TCLDccG80M5GeqGw inline hr=false]</p><p><span style=\"float:right\"><em>Source: Pathfinder Secrets of Magic</em></span></p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Interfering Surge] <span style=\"float:right\">Level 12</span></h2><ul class=\"tags paizo-style\"><li class=\"tag traits\">archetype</li></ul><p>@Embed[Compendium.pf2e.feats-srd.Item.aWHOcGLA7AhX4xpm inline hr=false]</p><p><span style=\"float:right\"><em>Source: Pathfinder Secrets of Magic</em></span></p>",
```

Comment:
Did you intentionally add "You can't select another dedication feat until you gain two other feats..."?